warehouse/router_identities.go

```diff
@@ -441,15 +438,15 @@ func (wh *HandleT) populateHistoricIdentities(ctx context.Context, warehouse mod
 
 		err = job.schemaHandle.fetchSchemaFromWarehouse(ctx, whManager)
 		if err != nil {
-			pkgLogger.Errorf(`[WH]: Failed fetching schema from warehouse: %v`, err)
+			r.logger.Errorf(`[WH]: Failed fetching schema from warehouse: %v`, err)
 			job.setUploadError(err, model.Aborted)
```

Comment:
How come we are ignoring this error? We're doing the same with other calls to `setUploadError` or `setUploadStatus`.